manual_jb/content/analyses.md

```diff
@@ -3,6 +3,7 @@
 1. [Bridge damage](analyses/bridge_dmg.md)
 2. [Building cluster recovery](analyses/building_cluster_recovery)
 3. [Building damage](analyses/building_dmg)
+4. [Building structural damage](analyses/building_structural_dmg)
```

Comment:
This should be after building non structural damage so it's in alphabetical order